Java/1-TwoSum.java

```diff
@@ -1,5 +1,5 @@
 /*
-
+     
```

Comment:
I don't think, we should have a space here

Reply:
Yeah Think so. But You have to think about it... Please Don't ignore....

Reply:
Did you test it out in the QA environment

Reply:
Yup.. and it works Perfect...

Reply:
Perfect, :)